Automattic/wp-calypso
# White-labeled plugin: Enable feature flag

Resolves https://github.com/Automattic/dotcom-forge/issues/9173

## Proposed Changes

* Enable the `migration-flow/enable-white-labeled-plugin` feature flag by default.
* Fix all tests that were previously related to Migrate Guru.
* Light cleanup and refactoring.

## Why are these changes being made?


* The white-labeled plugin is now on wp.org, so we're switching to installing that instead of Migrate Guru.

## Testing Instructions

* Switch to PR, navigate to `/setup/migration-signup`.
* Go through the flow to migrate a site. This includes having a site to migrate (you can use jurassic.ninja), checking out the business plan, following the migration instructions, and waiting for the provisioning steps to complete.
* Make sure the plugin is installed and the migration key can be retrieved.
* Follow the instructions and migrate the website using the plugin.
* Test the other migration flows:
  * `/setup/hosted-site-migration`
  * `/start` -> `Import existing content or website` -> `Migrate Site` -> `I'll do it myself`

https://github.com/user-attachments/assets/94bfbe30-adfb-402a-ac5f-bce50babdd97

## Pre-merge Checklist
